model/float/riscv_float_common.sail

```diff
@@ -0,0 +1,80 @@
+/*=======================================================================================*/
```

Comment:
Oh btw we shortened this massive copyright thing! If you pull master and rebase you'll see all the other files have much shorter messages now. Can you copy that here and in the other file?